lore/settings.py

```diff
@@ -353,5 +353,16 @@ def get_var(name, default):
     }
 }
 HAYSTACK_SIGNAL_PROCESSOR = 'haystack.signals.RealtimeSignalProcessor'
-ALLOW_CACHING = get_var("ALLOW_CACHING", get_var("ALLOW_CACHING", False))
+
 XANALYTICS_URL = get_var('XANALYTICS_URL', "")
+
+CACHES = {
+    "default": {
+        "BACKEND": get_var(
+            "CACHE_BACKEND", "django.core.cache.backends.locmem.LocMemCache")
```

Comment:
sounds good, while we're at it, do we even need to define the default cache at all?